.pre-commit-config.yaml

```diff
@@ -6,3 +6,15 @@ repos:
     -   id: end-of-file-fixer
     -   id: check-merge-conflict
     -   id: flake8
+
+-   repo: https://github.com/PyCQA/bandit
```

Comment:
One useful hook that will perform static code analysis, searching for bug and vulnerabilities. More information on the [official website](https://bandit.readthedocs.io/en/latest/).

Reply:
Hmm. From looking at the bandit site, it seems like the checks it performs are mostly geared toward web applications. I'm not sure the checks it performs make sense for a library like zipline. Is there a clear kind of error that you think this would catch/prevent in zipline? I think I'd prefer not to add this hook for now.

Reply:
I see, I think the majority of the tests are web security so I understand your point. I can remove this hook.